Add options to add more attribute to xliff template

vanvo1605 opened the pull request.

t2ym commented:
@vanvo1605 Can you add tests to your changes? Coverage falls down by -6.77% with your changes due to the missing coverage in your added codes below.  Thank you.

https://coveralls.io/builds/17441387/source?filename=xliff-conv.js#L680

### Note:
- Failures of tests due to the incorrect Sauce Labs credentials in your pull requests should be fixed automatically after the changes are merged into the master branch.

vanvo1605 commented:
Hi @t2ym, I updated something and added test for changes. Could you please help to review it?
https://github.com/t2ym/xliff-conv/pull/29/commits/c518af2f357b34ab732adc597400772d2068b7af
Thanks.

vanvo1605 commented:
Thank you!

Vào 04:43 PM T.4, 20 Th6 2018 Tetsuya Mori <notifications@github.com> đã
viết:

> Merged #29 <https://github.com/t2ym/xliff-conv/pull/29>.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/t2ym/xliff-conv/pull/29#event-1690675784>, or mute
> the thread
> <https://github.com/notifications/unsubscribe-auth/ATX9KRBxefy46-r9Kl7BwJzpakBKtEY9ks5t-hlPgaJpZM4Uj8xo>
> .
>
